requirements/extras/cassandra.txt

```diff
@@ -1 +1 @@
-cassandra-driver<3.21.0
+cassandra-driver~=3.24.0
```

Comment:
Are we sure we want to pin on 3.24? Why?

Reply:
Thank you for asking: this led me to a deeper investigation on a few warnings one gets when creating the `Session` object (aka connection to Cassandra) which I'm currently diligently mapping with the aim of providing a bit of guidance in the docs.

My (provisional) stance is that indeed 3.25 (currently the latest) is as good as 3.24 - and that actually 3.* would be fine.
I would forbid 4.* for the time being since, according to a warning one sees now, it will require an additional argument to the session constructor that is now optional - an argument I plan to cover in the addition to the docs anyway.
(Note: Cassandra drivers 4.* do not exist yet.)

I will confirm on this thread once I finish my checks.

Reply:
So I'm leaning on `cassandra-driver>=3.24,<4` at the moment.

Also, I forgot to mention: 3.23 does not seem to be able to connect to AstraDB yet.

Reply:
My latest commit has `>=3.24,<4` and in the docs I cover the additional configuration needed to silence all warnings from the driver doing guesswork/negotiation on client side. I think that's all good now.

Reply:
Also I extended the test coverage for the both cluster-creation code pathways (I believe the coverage should now be non-decreasing).